receiver/statsdreceiver/protocol/statsd_parser.go

```diff
@@ -168,6 +168,8 @@ func (p *StatsDParser) Initialize(enableMetricType bool, isMonotonicCounter bool
 		case TimingTypeName, TimingAltTypeName:
 			p.timerEvents.method = eachMap.ObserverType
 			p.timerEvents.histogramConfig = expoHistogramConfig(eachMap.Histogram)
+		case CounterTypeName:
+		case GaugeTypeName:
```

Comment:
I personally object to this level of "linting". Can these two branches be a single `default` branch instead? (What is actually improving here?)